From 5f8e10b1ed186f79d3561a5163a2ae4821fc2305 Mon Sep 17 00:00:00 2001
Message-ID: <5f8e10b1ed186f79d3561a5163a2ae4821fc2305.1724198118.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1724198118.git.sharlatanus@gmail.com>
References: <cover.1724198118.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 31 Jul 2024 08:43:40 +0100
Subject: [PATCH 15/32] gnu: python-reproject: Update to 0.14.0.

* gnu/packages/astronomy.scm (python-reproject): Update to 0.14.0.
[propagated-inputs]: Remove python-click.
[native-inputs]: Remove python-pytest; add python-asdf, python-gwcs and
python-pyvo.

Change-Id: If3434c59635fb818923f1bc7e292d494166d50be
---
 gnu/packages/astronomy.scm | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index e505b4abcb..a9dd4360c4 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -3119,13 +3119,13 @@ (define-public python-regularizepsf
 (define-public python-reproject
   (package
     (name "python-reproject")
-    (version "0.13.1")
+    (version "0.14.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "reproject" version))
        (sha256
-        (base32 "1xk809h2j3zj37l5lx4l87zanf9zxbxajcrff2b2f2i0jpnmxqv3"))))
+        (base32 "1h5v74da32ai9dkhcdn3xx4h05awlxhhnrsdjj1sb644c1vlyg6x"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -3162,7 +3162,6 @@ (define-public python-reproject
      (list python-asdf
            python-astropy
            python-astropy-healpix
-           python-click
            python-cloudpickle
            python-dask
            python-fsspec
@@ -3175,9 +3174,11 @@ (define-public python-reproject
     (native-inputs
      (list python-cython-3
            python-extension-helpers
-           python-pytest
+           python-asdf
+           python-gwcs
            python-pytest-astropy
            python-pytest-xdist
+           python-pyvo
            ;; python-sunpy ; circular dependencies, test optional
            python-setuptools-scm))
     (home-page "https://reproject.readthedocs.io")
-- 
2.41.0

